js/navigation.jquery.js

```diff
@@ -304,6 +304,90 @@ class Navigation {
     // Handle in-page bookmarks.
     if (document.location.hash) this.hashChanged();
     $(window).on('hashchange', $.proxy(this.hashChanged, this));
+
+    // Accessibility
+    this.$container.on('keydown', (ev) => {
```

Comment:
Can we have this behaviour be optional but enabled by default?

Reply:
How do you want the opt out to work, `data-` attribute on the `nav` element or something?

Reply:
data on .id7-navigation, which is how the other optional functionality works (e.g. fixedHeader, fixedNav etc)

Reply:
Done with `data-no-accessibility="true"`